style.css

```diff
@@ -101,24 +88,742 @@ body {
   font-size: 16px;
   line-height: 24px;
   margin-right: 24px;
+}
+
+.main-container {
+  background-image: url("image/illustration-header-mobile.png");
+  background-repeat: no-repeat;
+  background-position: right bottom;
+  background-position-y: 200px;
+  padding-bottom: 0;
+  height: 580px;
+  background-color: #1c1a19;
+}
+
+.social-connect {
+  margin-left: 10px;
+  padding: 10px;
+}
+
+.work-section {
+  height: 880px;
+}
+
+.work-title h2 {
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  color: #1c1a19;
+  padding-top: 120px;
+  padding-top: 0;
+}
+
+.line {
+  width: 327px;
+  margin-right: 96px;
+  margin-bottom: 80px;
+}
+
+.underline {
+  border-bottom: 1px solid #fff;
+  margin-left: 24px;
+  margin-right: 24px;
+  position: relative;
+  margin-bottom: 200px;
+  top: 300px;
+}
+
+.main-wrap {
+  background: #fff;
+  margin-top: 8rem;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.multi-post {
+  height: 147px;
+  line-height: 24px;
+
+  /* width: 375px; */
```

Comment:
- [ ] [REQUIRED]  Please remove any line of code that you are not using as a good practice of making your code clean.😊